pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -387,10 +429,28 @@ func (og *operationGenerator) GenerateDetachVolumeFunc(
 		if err != nil {
 			return volumetypes.GeneratedOperations{}, volumeToDetach.GenerateErrorDetailed("DetachVolume.SplitUniqueName failed", err)
 		}
-		attachableVolumePlugin, err = og.volumePluginMgr.FindAttachablePluginByName(pluginName)
-		if err != nil {
-			return volumetypes.GeneratedOperations{}, volumeToDetach.GenerateErrorDetailed("DetachVolume.FindAttachablePluginBySpec failed", err)
+
+		// TODO(dyzz): This case can't distinguish between PV and In-line which is necessary because
+		// if it was PV it may have been migrated, but the same plugin with in-line may not have been.
+		// Suggestions welcome...
```

Comment:
It is hard to do that since CSI In-line volume support is also going alpha this quarter so the dependency there is kind of tight. I guess during Q4 alpha of migration this specific functionality can be broken and fixed in 1.14. It is a non-standard codepath anyway (only hit on A/D controller restart when pod was deleted in meantime).
/cc @saad-ali

Reply:
Coupling this feature with the CSI inline feature will slow everything down. This case is an edge case. Let's document it as a known issue with the alpha release.

Reply:
It's ok for alpha to develop them separately, but IMO inline volumes should go beta and GA before or together with the migration. It's not full feature if you migrate just the easier part :-).

Reply:
@jsafrane we plan to have in-line volumes migrated by Q1 for alpha as well. Ideally they will go Beta together Q2